src/bowling_SC.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class bowlingGame {
+
+  constructor() {
+    this.rolls = [[], [], [], [], [], [], [], [], [], []];
+    this.runningScore = 0;
+    this.frameIndex = 1;
+    this.rollIndex = 0;
```

Comment:
It doesn't look like we ever reassign a new value to this variable - should this be a constant?